.circleci/config.yml

```diff
@@ -361,7 +361,7 @@ jobs:
           name: Set cluster
           command: |
             sudo pip install -r requirements.pip
-            mkdocs gh-deploy -m "[ci skip] Updated documentation."
+            mike deploy -u $(cat .version) latest -m "[ci skip] Updated documentation."
```

Comment:
What is this mike?

Reply:
https://github.com/jimporter/mike

Reply:
yes, that mike

Reply:
also added to the requirements.pip

Reply:
This basically allows you to deploy several versions of the documentation in parallel to the `gh-pages` branch (while `mkdocs`, on the other hand, always overwrites the files in the branch). :+1: